README.md

```diff
@@ -26,10 +31,12 @@ Pressure/temperature sensing is supported for the following pressure sensors:
 
 Select the IMU in use by editing libraries/RTIMULib/RTIMULibDefs.h and uncommenting one of the supported IMUs like this:
 
-	#define MPU9150_68                      // MPU9150 at address 0x68
+	//#define MPU9150_68                      // MPU9150 at address 0x68
 	//#define MPU9150_69                      // MPU9150 at address 0x69
 	//#define MPU9250_68                      // MPU9250 at address 0x68
 	//#define MPU9250_69                      // MPU9250 at address 0x69
+	#define MPU9255_68                        // MPU9255 at address 0x68
```

Comment:
I don't want this to break mine when I pull it, otherwise I'm good with this

Reply:
Ok no problems, perfect!